packages/sprinkles/src/createUtils.ts

```diff
@@ -1,4 +1,4 @@
-import { addRecipe } from '@vanilla-extract/css/recipe';
+import { addFunctionSerializer } from '@vanilla-extract/css/functionSerializer';
```

Comment:
Not sure we should change this as it'll break for consumers running older versions of the integration package for no benefit.

Reply:
Yeah this needs to stay as `addRecipe` for the time being